Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN apk add --no-cache --quiet \
   bash \
   curl \
   dumb-init \
-  git && \
+  git mercurial && \
```

Comment:
What is mercurial?

Reply:
Mercurial provides the lib `hg` which is used by codecov's upload script-- however we did not need it in the end.